### PR TITLE
Add Level Order Traversal For Binary Tree (C++)

### LINKED ISSUE #192: Level Order Traversal of Binary Tree
I'd like to work ok this as a participant of GSSOC'21. Kindly assign it to me.

### PR DESCRIPTION
**Pull request by AnjaliPatle**
Closes #192

### CODE REVIEW

File: Code/C++/level_order_traversal_binary_tree.cpp [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+// present in the queue, this will be the count of the nodes in the particular level. Then do the same operations as mentioned above to the
+// Nodes of this level. After we have visited every node of this level, we print a newline and start the next iteration for the next level.
+
+
+#include <iostream>
+#include <bits/stdc++.h>
+using namespace std;
+
+// A Binary Tree Node
+struct Node
```
Comment (rudrakshi99) [inline Code/C++/level_order_traversal_binary_tree.cpp:33]: Don't use struct in c++. Use class.

File: Code/C++/level_order_traversal_binary_tree.cpp [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    root->right->right = createNode(16);
+    
+    cout << "Level Order traversal of binary tree is: \n";
+    PrintLevelOrder(root);
+    cout<<endl;
+    
+    cout<<"Level Order Traversal with of each level individually is: \n";
+    PrintLevelOrderIndividually(root);
+    
+    return 0;
```
Comment (rudrakshi99) [inline Code/C++/level_order_traversal_binary_tree.cpp:130]: Add at least 2 test cases. Also, add time and space complexities.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Comment by AnjaliPatle** [comment]
Hey @rudrakshi99 , I've added testcases and time and space complexities. Please let me know if there are any more changes.

**Comment by rudrakshi99** [comment]
Add the 2 test cases in the comments also at the end with input and output. Do refer to our code folder, you will get an idea.

File: Code/C++/level_order_traversal_binary_tree.cpp [hunk 3]
```diff
@@ -0,0 +1,131 @@
+//LEVEL ORDER TRAVERSAL OF BINARY TREE
```
Comment (rudrakshi99) [inline Code/C++/level_order_traversal_binary_tree.cpp:1]: use multi comment.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Comment by AnjaliPatle** [comment]
@rudrakshi99  I've made the required changes.

**Comment by rudrakshi99** [comment]
Just replace `null` with `nullptr` then we are good to merge.

**Comment by AnjaliPatle** [comment]
I have made the required changes. Kindly review it.

**Review by OjusWiZard — APPROVED** [review]

**Review by rudrakshi99 — APPROVED** [review]

**Pull request merged**